contextily/place.py

```diff
@@ -8,6 +8,11 @@
 from . import providers
 from ._providers import TileProvider
 
+# Set user ID for Nominatim
+rng = np.random.default_rng()
```

Comment:
I think `default_rng` is only available for recent numpy versions. Since we don't really care here which generator is exactly used, I would do `np.random.randint(..)` instead, that will work accross versions